src/translations/locales/en.tsx

```diff
@@ -1,3 +1,3 @@
 export const TRANSLATIONS_EN = {
-  hello: 'Hello world',
+  button: 'Add to cart',
```

Comment:
Instead of having a key button, it's best to have one like addToCart because we're going to have many buttons with different content/translations in them